CMakeLists.txt

```diff
@@ -164,15 +173,11 @@ endif()
 if(ACTS_BUILD_EXAMPLES_PYTHIA8)
   find_package(Pythia8 REQUIRED)
 endif()
+# other dependencies
 if(ACTS_BUILD_DOCS)
   find_package(Doxygen 1.8.11 REQUIRED)
   find_package(Sphinx REQUIRED)
 endif()
-if(ACTS_BUILD_CUDA_PLUGIN)
```

Comment:
As far as I can see, this was removed and not added back, which is likely to cause problems.

Reply:
No, this is still in. It was moved to the top of the selection to keep alphabetic order.